edward/stats/distributions.py

```diff
@@ -9,114 +9,83 @@ class Distribution:
+            If univariate distribution, can be a scalar, vector, or matrix.
+            If multivariate distribution, can be a vector or matrix.
 
-        params : float, np.array, or tf.Tensor
+        params : np.array or tf.Tensor
             scalar unless documented otherwise
 
         Returns
         -------
         tf.Tensor
-            If univariate distribution, returns a tensor of same shape
-            as input. If multivariate distribution, returns a tensor
-            of shape[:-1] from input: the outer dimension representing
-            the multivariate dimension is collapsed.
-
-        Notes
-        -----
-        x as a 3d or higher tensor is not guaranteed to be supported
-        (for either univariate or multivariate distribution).
+            For univariate distributions, returns a scalar, vector, or
+            matrix corresponding to the size of input. For
+            multivariate distributions, returns a scalar if vector
+            input and vector if matrix input, where each element in
+            the vector evaluates a row in the matrix.
         """
         raise NotImplementedError()
 
     def entropy(self):
         """
         Parameters
         ---------
-        params : float, np.array, or tf.Tensor
-            If univariate distribution, can be a scalar, vector, or
-            matrix.
+        params : np.array or tf.Tensor
+            If univariate distribution, can be a scalar or vector.
             If multivariate distribution, can be a vector or matrix.
 
         Returns
         -------
         tf.Tensor
-            If univariate distribution, returns a tensor of same
-            shape as input.
-            If multivariate distribution, returns a tensor of
-            shape[-1] from input: the outer dimension representing the
-            multivariate dimension is collapsed.
+            For univariate distributions, returns a scalar or vector
+            corresponding to the size of input. For multivariate
+            distributions, returns a scalar if vector input and vector
+            if matrix input, where each element in the vector
+            evaluates a row in the matrix.
 
         Notes
         -----
-        The equivalent method in SciPy is not guaranteed to be
-        supported for vector inputs for univariate distributions and
-        matrix inputs for multivariate distributions. This is.
-
-        params as a 3d or higher tensor is not guaranteed to be
-        supported (for either univariate or multivariate
-        distribution).
+        SciPy doesn't always enable vector inputs for
+        univariate distributions or matrix inputs for multivariate
+        distributions. This does.
         """
         raise NotImplementedError()
 
 class Bernoulli:
+    """Bernoulli distribution
+    """    
     def rvs(self, p, size=1):
-        """
+        """Random variable generator
+
+        Parameters
+        ----------
+        p : float
```

Comment:
Following the template description, the type is actually `float or np.array`. Same goes with all the other distribution methods with `rvs()`.